CODEOWNERS

```
@@ -236,7 +236,9 @@ Kconfig*                                  @tejlmand
 /snippets/nrf91-modem-trace-ext-flash/    @nrfconnect/ncs-cia
 /snippets/nrf91-modem-trace-uart/         @eivindj-nordic
 /snippets/tfm-enable-share-uart/          @nrfconnect/ncs-cia
-/snippets/nrf70-debug/                    @krish2718 @sachinthegreen
+/snippets/nrf70-driver-verbose-debug/     @krish2718 @sachinthegreen
+/snippets/nrf70-driver-debug/             @krish2718 @sachinthegreen
+/snippets/nrf70-supplicant-debug/         @krish2718 @sachinthegreen
```

```suggestion
/snippets/wpa-supplicant-debug/         @krish2718 @sachinthegreen
```
This is common code.